ape_trezor/client.py

```diff
@@ -85,31 +85,27 @@ def sign_personal_message(self, message: bytes) -> Tuple[int, bytes, bytes]:
         to validate the message data.
         """
         ethereum_message_signature = ethereum.sign_message(
-            self._client, parse_hdpath(self._account_hd_path.path), message
+            self.client, parse_hdpath(self._account_hd_path.path), message
         )
 
         return extract_signature_vrs_bytes(signature_bytes=ethereum_message_signature.signature)
 
-    # TODO: Uncomment when Trezor has released the EIP 712 update
```

Comment:
if this is released now, I would expect us to have to bump the version number of `trezor` in the `setup.py` to be able to use it?